### PR TITLE
Open links in an in-app browser instead of safari

### PR DESCRIPTION
**Pull request by mplorentz**
kind of related to #294 - I'm just trying to make the thread view a better experience

### CODE REVIEW

File: Nos/Views/RelayView.swift [hunk 1]
```diff
@@ -32,32 +32,59 @@ struct RelayView: View {
...
+                    Task {
+                        for index in indexes {
+                            let relay = relays[index]
+                            await relayService.closeConnection(to: relay)
+                            analytics.removed(relay)
+                            author.remove(relay: relay)
+                            viewContext.delete(relay)
+                        }
+                        
+                        try! viewContext.save()
```
Comment (martindsq) [inline Nos/Views/RelayView.swift:50]: Shouldn't force try this database save operation and should show the error in an alert instead

File: Nos.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1622,10 +1632,10 @@
 				"INFOPLIST_KEY_UIStatusBarStyle[sdk=iphonesimulator*]" = UIStatusBarStyleDefault;
 				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad = "UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
 				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone = "UIInterfaceOrientationPortrait UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
-				IPHONEOS_DEPLOYMENT_TARGET = 16.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
```
Comment (martindsq) [inline Nos.xcodeproj/project.pbxproj:1635]: I did notice that the deployment target was 16.2 which seemed too recent for me, but I didn't know if it was intended or not. This kind of changes should be done in a distinct PR so can I know for sure if there are intended to be merged or not. The same for macos deployment target.

**Review by martindsq — CHANGES_REQUESTED** [review]

File: .xcode-version [hunk 3]
```diff
@@ -1 +1 @@
-14.2
+14.3
```
Comment (martindsq) [inline .xcode-version:1]: This should be also in a distinct PR

**Comment by mplorentz** [comment]
@martindsq sorry I think I had the wrong target branch on this. The diff is cleaner now.

**Review by martindsq — APPROVED** [review]

**Comment by martindsq** [comment]
It's nice now!

**Pull request merged**